package.json

```diff
@@ -56,6 +56,7 @@
     "@babel/runtime-corejs3": "7.18.9",
     "@nrwl/nx-cloud": "14.3.0",
     "@playwright/test": "1.25.1",
```

Comment:
We only officially support playwright v1.19 because newer versions do not work with Firefox. If you stick with chromium, you should be okay but keep an eye out for playwright errors if you continue to update the version.

This story will improve in the coming weeks when we update our Firefox fork to the latest ESR (102.2).